### DISCUSSION TITLE
Integration with Existing Angular app

### DISCUSSION THREAD

**Discussion by poojakedar** [neomjs/neo#3383] [Q&A]
Hi Team,

Your app looks impressive. Actually, we are also delivering an app with multi window feature. Any chance that we can integrate it with our existing app. We are finding it difficult to send data across the browser window, currently we use the localStorage but it does not work perfectly all the time.  But I see Neomj lib fulfills this purpose.

Kindly share your thoughts on how to move forward.

**Comment by hmachefe** [comment]
I can't agree more. Stunning.

Is there any plan to integrate neo mjs in Angular projects ? Any roadmap ?
How hard would it be to make such different ecosystems (Anguar, Neo mjs) coexist ?

Hugo
